lldb/test/API/python_api/interpreter/TestCommandOverrideCallback.py

```diff
@@ -0,0 +1,30 @@
+from typing_extensions import override
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+
+class CommandOverrideCallback(TestBase):
+    def setUp(self):
+        TestBase.setUp(self)
+        self.line = line_number("main.c", "Hello world.")
+
+    def test_command_override_callback(self):
+        self.build()
+        exe = self.getBuildArtifact("a.out")
+
+        # Create a target by the debugger.
```

Comment:
I don't think this comment is helpful because it's telling me what the code is doing, not why it's doing it. I'd suggest removing it because pretty much every test does this.